README.md

```diff
@@ -8,7 +8,9 @@ Gem to add contacts, deals and organizations to Agendor CRM
 
 ### Usage
 
-You can get the list of all allowed resources attributes at https://api.agendor.com.br 
+You can get the list of all allowed resources attributes at https://api.agendor.com.br
+
+Parameters `username` and `password` are optional. Token will be used when present (not nil or '').
```

Comment:
Essa frase makes no sense:
`Token will be used when present (not nil or '')` 

O Token vai estar presente quando for `not nil` ou **NÃO** for uma string vazia.